app/views/api/v2/parameters/base.json.rabl

```diff
@@ -1,7 +1,3 @@
 object @parameter
 
-attributes :id, :name, :parameter_type, :associated_type, :hidden_value?
-
-node do
-  partial("api/v2/common/show_hidden", :locals => { :value => :value }, :object => @object)
```

Comment:
What was the reason to remove this node? We should not change the API,
customers can have their own infra on the top pf our API and this is a breaking change